src/Client/ForSubject.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace eLife\ApiSdk\Client;
+
+trait ForSubject
+{
+    private $subjectsQuery = [];
+
+    final public function forSubject(string ...$subjectId) : self
+    {
+        $clone = clone $this;
+
+        $clone->subjectsQuery = array_unique(array_merge($this->subjectsQuery, $subjectId));
+
+        if ($clone->subjectsQuery !== $this->subjectsQuery) {
+            $clone->count = null;
```

Comment:
Probably want to declare `$count` on the trait.

Reply:
Or cater for https://github.com/elifesciences/api-sdk-php/pull/204#discussion_r400279494 too by calling an abstract method.